### DISCUSSION TITLE
Step-by-Step Guide: Setting Up OpenXPKI for Secure Digital Certificates in Linux

### DISCUSSION THREAD

**Discussion by nbaglietto** [community/community#110656] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hola hoy estaba viendo tu guia (muy buena por cierto) y se me presento un problema con los perfiles de los usuariosya que caop y raop no funcionan no puedo loguearme al sistema. te agradeceria mucho que me explicaras como cambiar el passwd o como eliminar esos perfiles y crear un perfil de administrador. Atte. Nahuel

**Comment by ettaboyle** [comment]
Hi @nbaglietto, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
